edx_solutions_api_integration/courses/views.py

```diff
@@ -1319,6 +1322,10 @@ def get_queryset(self):
             users = users.prefetch_related(
                 Prefetch('course_groups', queryset=CourseUserGroup.objects.filter(course_id=self.course_key))
             )
+        try:
+            order_by_field = User._meta.get_field(self.request.query_params.get('order_by', 'id'))
```

Comment:
@Agrendalath please ignore, I _really_ need sleep. Jeez.

Reply:
Ugh, I was about to comment on this.

Reply:
They should actually support this case, but oh well.

Reply:
Ah, what I mean was that query params are retrieved at the top for all endpoints and this is inconsistent with it.

Reply:
Oh, I see. It doesn't work this way because `get_field` returns a non-string type, and Django doesn't handle non-strings in `order_by`. They should support the field type or whatever they return with `get_field`.